cypress/integration/searchTest.js

```diff
@@ -0,0 +1,17 @@
+import HomePage from '../../src/pages/homePage'
+import ShopPage from '../../src/pages/shopPage'
+import * as shopData from '../../testData/shopData'
+
+describe("Perform search for iPhone11 2019 16GB metall", ()=>{
+    
+    it("Search for iPhone11", ()=>{
+        HomePage.openUrl(shopData.shopPhonesUrl)
+        ShopPage.selectBrand()
+        ShopPage.clickAdvancedSearch()
+        ShopPage.selectMaterial()
+        ShopPage.fillInRam(shopData.ramSize)
+        ShopPage.selectYear()
+        ShopPage.clickShow()
```

Comment:
One of the main approaches in PageObject pattern is a possibility to return pages in methods.
Using this approach you can avoid duplication of `ShopPage` in lines 9-14.
Just add a `return` statement to methods where it's necessary in `src/pages/shopPage.js` and you will get a possibility to chain methods:
```
ShopPage.selectBrand()
        .clickAdvancedSearch()
        .selectMaterial()
        .fillInRam(shopData.ramSize)
        .selectYear()
        .clickShow()
```